src/matching/Matching_generic.mli

```diff
@@ -10,6 +10,7 @@ type tin = {
   deref_sym_vals : int;
       (** Counts the number of times that we "follow" symbollically propagated
     * values. This is bound to prevent potential infinite loops. *)
+  wildcard_imports : AST_generic.ident list list;
```

Comment:
`AST_generic.dotted_ident list` and some doc string explaining the purpose of this list.